python/paddle/tensor/math.py

```diff
@@ -6677,7 +6677,7 @@ def i0_(x, name=None):
     Please refer to :ref:`api_paddle_i0`.
     """
 
-    if in_dynamic_mode():
+    if in_dynamic_or_pir_mode():
```

Comment:
i0_ 函数被装饰器 `@inplace_apis_in_dygraph_only` 修饰，只能运行在动态图模式下；而我们的 pir 属于静态图，所以这里不需要改动，复原一下吧~
```suggestion
    if in_dynamic_mode():
```

Reply:
好的，done

Reply:
好的，done